nominal/__init__.py

```diff
@@ -1,13 +1,48 @@
-import os
+from .nominal import (
+    set_default_connection,
+    get_default_connection,
+    upload_dataset_from_pandas,
+    upload_dataset_from_polars,
+    upload_dataset,
+    get_dataset_by_rid,
+    update_dataset,
+    create_run,
+    get_run_by_rid,
+    search_runs,
+    update_run,
+    add_dataset_to_run,
+    list_datasets_for_run,
+    add_attachment_to_run,
+    list_attachments_for_run,
+    upload_attachment,
+    get_attachment_by_rid,
+    update_attachment,
+    save_attachment,
+)
+from .sdk import Dataset, Run, Attachment, NominalClient
 
-# Allows:
-# import nominal as nm
-# nm.cloud ...
-# nm.data ...
-from . import cloud as cloud
-from . import data as data
-from .nominal import Dataset as Dataset
-from .nominal import Ingest as Ingest
-from .nominal import Run as Run
-
-os.environ["NOMINAL_BASE_URL"] = "https://api-staging.gov.nominal.io/api"
+__all__ = [
+    "set_default_connection",
+    "get_default_connection",
+    "upload_dataset_from_pandas",
+    "upload_dataset_from_polars",
+    "upload_dataset",
+    "get_dataset_by_rid",
```

Comment:
should this be a "high level" method? 🤔 

if so, I think it should be thought of as the "primary" way to get datasets - maybe `find_dataset` would be the name for like, search by name or channels or something else

Reply:
you're suggesting `get_dataset_by_rid()` -> `get_dataset()` since it's the main way (so the verbosity doesn't help disambiguate), correct?

Reply:
so specifically this could be called `get_dataset` & ignore the "by_rid" bit if there's no other way to get a single dataset